docs/team/Peng Xuan Chang.adoc

```diff
@@ -50,7 +53,7 @@ This application was built upon an existing code base for an address book for th
 
 include::../UserGuide.adoc[tag=listcommand]
 
-include::../UserGuide.adoc[tag=dataencryption]
+include::../UserGuide.adoc[tag=undoredo]
```

Comment:
Please update undo/redo section in the UG. Screen shots should have annotations to make it easier for the reader to know what is the focus. Do listen to the UGDG consultation recording.

Reply:
Thanks for reminding 👍 . I haven't touched UndoRedo sections yet (since I haven't pulled from team repo to update the populated sample data). I'll issue another PR to update these sections and add in annotated screenshots. Cheers :)